README.md

```diff
@@ -0,0 +1,30 @@
+# Custom Alpine Image for virtual machines
+
+This repository provides a simple and efficient way to build an Alpine-based virtual machine image.
+
+## Building the Image
+
+Use the `build.sh` script to generate the image:
+
+```sh
+./build.sh
+```
+
+Options:
+- `-s, --size <size>`: Specify the image size (default: 1500M).
+- `-f, --dockerfile <path>`: Specify a custom Dockerfile (default: `./Dockerfile`).
+- `-o, --output <file>`: Specify the output image file (default: `image.ext2`).
+
+For additional details on creating and customizing ext2 images for use with CheerpX, refer to the [custom disk images](https://cheerpx.io/docs/guides/custom-images) guide in the CheerpX documentation.
+
+## License
```

Comment:
The license must be in a separate file called LICENSE, like we do in other repos. There is no CX deployment connected to this repo.